app/controllers/apo_controller.rb

```diff
@@ -3,10 +3,25 @@
 
 class ApoController < ApplicationController
 
-  before_filter :create_obj, :except => [:register, :is_valid_role_list_endpoint, :spreadsheet_template]
-  after_action :save_and_index, :only => [:delete_collection, :delete_collection, :add_collection, :update_title, :update_creative_commons, :update_use, :update_copyright, :update_default_object_rights, :add_roleplayer, :update_desc_metadata, :delete_role, :register_collection]
-
-  DEFAULT_MANAGER_WORKGROUPS = ['sdr:developer', 'sdr:service-manager', 'sdr:metadata-staff']
+  before_filter :create_obj, :except => [
+    :is_valid_role_list_endpoint,
+    :register,
+    :spreadsheet_template
+  ]
+  after_action :save_and_index, :only => [
+    :add_roleplayer,
+    :add_collection, :delete_collection,
+    :update_copyright, :update_creative_commons,
+    :update_default_object_rights, :update_desc_metadata,
+    :update_title, :update_use,
+    :delete_role,
+    :register_collection
+  ]
+
+  # These manager workgroups are specific to APO permissions and they are
+  # distinct from the repository-wide permissions in User.MANAGER_GROUPS
+  # This is used in app/views/apo/register.html.erb
+  DEFAULT_MANAGER_WORKGROUPS = %w(sdr:developer sdr:service-manager sdr:metadata-staff).freeze
```

Comment:
Hound.ci recommended the `%w()` style and the `.freeze` for a constant with a mutable object.